modules/govuk_rabbitmq/manifests/consumer.pp

```diff
@@ -48,9 +61,9 @@
       ensure => $ensure,
       type   => $exchange_type,
     }
-    $write_permission = "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})$"
+    $write_permission = "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})\$"
```

Comment:
whats the extra slash for here? Does this tell rabbitmq to match a literal $ in a name?